Formula/liquidprompt.rb

```diff
@@ -29,6 +29,6 @@ def caveats
   test do
     liquidprompt = "#{HOMEBREW_PREFIX}/share/liquidprompt"
     output = shell_output("/bin/sh #{liquidprompt} 2>&1")
-    assert_match "add-zsh-hook: command not found", output
+    assert_match "#{liquidprompt}: line 25: return: can only `return' from a function or sourced script", output
```

Comment:
This is not a good test, as you're testing whether `/bin/sh` is reading commands correctly. So this is not a test of `liquidprompt` at all.

Reply:
I agree, but the [original test](https://github.com/Homebrew/homebrew-core/commit/91a45347ef8efdc7ee2fa5247ceda6b04c634594) was dropped in by someone unfamiliar with `liquidprompt` and it was accepted. I just updated the assertion in accord with the newest version of the tool.